OpenScienceJournal/whistlepunk_library/src/main/java/com/google/android/apps/forscience/whistlepunk/actionarea/ActionAreaItemView.java

```diff
@@ -61,9 +61,12 @@ protected void setActionAreaItem(
     }
   }
 
-  protected void updateView(Context context, int style) {
+  protected void updateView(Context context, int optionalStyle) {
     if (actionAreaItem != null) {
-      ContextThemeWrapper wrapper = new ContextThemeWrapper(context, style);
+      ContextThemeWrapper wrapper = new ContextThemeWrapper(context, context.getTheme());
+      if (optionalStyle != -1) {
```

Comment:
Can we make -1 a constant somewhere?

Reply:
Just added an additional commit that turns -1 into a constant